### PR TITLE
Sprint 3

### PR DESCRIPTION
**Pull request by drodr518**
added Book, Service and Repo

### CODE REVIEW

File: src/main/java/com/bookstore/bookstore/services/BookServiceImpl.java [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+
+import com.bookstore.bookstore.models.Book;
+
+import java.util.List;
+
+public class BookServiceImpl implements BookService
+{
+    @Override
+    public List<Book> searchTitle(String title) {
+        return null;
```
Comment (lorlandop30) [inline src/main/java/com/bookstore/bookstore/services/BookServiceImpl.java:15]: Why returning null?

File: src/main/java/com/bookstore/bookstore/services/BookServiceImpl.java [hunk 2]
```diff
@@ -0,0 +1,28 @@
+package com.bookstore.bookstore.services;
+
+import com.bookstore.bookstore.models.Book;
+
+import java.util.List;
+
```
Comment (lorlandop30) [inline src/main/java/com/bookstore/bookstore/services/BookServiceImpl.java:6]: @Service 

if not , it won't work

File: src/main/java/com/bookstore/bookstore/controllers/BookController.java [hunk 3]
```diff
@@ -0,0 +1,7 @@
+package com.bookstore.bookstore.controllers;
+
+import org.springframework.stereotype.Controller;
+
+@Controller
```
Comment (lorlandop30) [inline src/main/java/com/bookstore/bookstore/controllers/BookController.java:15]: are you still working on this?

**Review by lorlandop30 — CHANGES_REQUESTED** [review]

File: src/main/java/com/bookstore/bookstore/services/BookServiceImpl.java (hunk 1 above)
Comment (drodr518) [reply src/main/java/com/bookstore/bookstore/services/BookServiceImpl.java:15]: Placeholder until I build out my data access layer

File: src/main/java/com/bookstore/bookstore/services/BookServiceImpl.java (hunk 2 above)
Comment (drodr518) [reply src/main/java/com/bookstore/bookstore/services/BookServiceImpl.java:6]: ok

**Pull request closed without merging**